models/attribute-type/src/attribute-boolean-type/attribute-boolean-type-draft/transformers.ts

```diff
@@ -0,0 +1,27 @@
+import { Transformer } from '@commercetools-test-data/core';
+import {
+  TAttributeBooleanTypeDraftGraphql,
+  TAttributeBooleanTypeDraft,
+} from '../types';
+
+const transformers = {
+  default: Transformer<TAttributeBooleanTypeDraft, TAttributeBooleanTypeDraft>(
+    'default',
+    {
+      buildFields: [],
+    }
+  ),
+  graphql: Transformer<
+    TAttributeBooleanTypeDraft,
+    TAttributeBooleanTypeDraftGraphql
+  >('graphql', {
+    buildFields: [],
+    replaceFields: ({ fields }) => ({
+      boolean: {
+        dummy: fields.name,
```

Comment:
Graphql expects this field oddly enough. I assumed passing along any string would be fine.

Reply:
I see this but what is it for??? (probably just shouting frustration into the void)

Reply:
Ah, I see that this field is nullable. I'm going to pass that instead

Reply:
All of which is to say, I have no idea what this field is for either 😆